test/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@ project(concurrencppTests LANGUAGES CXX)
 
 include(../cmake/coroutineOptions.cmake)
 
+if(CMAKE_CONFIGURATION_TYPES)
```

Comment:
This variable is not an indication whether the generator is multi-config or not, but this bit of code has no business to do anything here. Like I said in a previous comment, this should be left to be decided by the person configuring the project, since there are many ways to solve this, it's harmful to make a decision that might not fit their preferred workflow. Only the CI workflow should be adjusted to append to `PATH` on Windows.

Reply:
I only partially agree. It should work by default, a user shouldn't have to figure out which arcane cmake thing they have to do to make it work. Moreover, this is specifically for the test project, not the main one, so it only affects individuals who are specifically building tests.

I changed it so that it will only set CMAKE_RUNTIME_OUTPUT_DIRECTORY if it isn't already defined (and I fixed the multi-config check).